android/app/src/main/java/host/exp/exponent/MainApplication.java

```diff
@@ -122,8 +122,15 @@ public String gcmSenderId() {
     return getString(R.string.gcm_defaultSenderId);
   }
 
+  // sdk > 32 or UNVERSIONED
   public static OkHttpClient.Builder okHttpClientBuilder(OkHttpClient.Builder builder) {
     // Customize/override OkHttp client here
     return builder;
   }
+
+  // sdk <= 32
+  public static expolib_v1.okhttp3.OkHttpClient.Builder okHttpPrefixedClientBuilder(expolib_v1.okhttp3.OkHttpClient.Builder builder) {
```

Comment:
We can remove this, as `MainApplication` class is only used in ejected apps and ejected app won't ever use `expolib_v1` from now on. 🙂